lib/questions.rb

```diff
@@ -110,12 +127,14 @@ def convert_array_to_a_hash(array)
 # . e.g. the array ['cat', 'dog', 'fish'] becomes
 # ['a', 'c', 'd', 'f', 'g', 'h', 'i', 'o', 's', 't']
 def get_all_letters_in_array_of_words(array)
+  array.map{|e|e.chars}.flatten.sort
```

Comment:
Space after closing `|` missing.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:chars` as an argument to `map` instead of a block.